src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/PublicationsAccordion.tsx

```diff
@@ -25,15 +25,17 @@ import {
   AccordionItemButton
```

Comment:
What does the PublicationsAccordion do? Do we still need it after we decided to just link to Europe PMC?

Reply:
I am not quite sure about that decision but i'm happy to clean it up in this PR.

Reply:
The updated XD (in https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-824) doesn't have a publications accordion

Reply:
Thanks! That's done.